.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -9,10 +9,16 @@ assignees: ''
 
 <!-- NOTE:
 
+Those information will be helpful for us to investigate the problem. So please, follow the steps we provided exactly when submitting your issue, and do not remove the issue template.
```

Comment:
"A good bug report should also have your configuration and build environment information, which are essential for us to investigate the problem. We've provided the following steps on how to attach the necessary information."

Sounds a bit more neutral.